### DISCUSSION TITLE
Multiple Interface assertion

### DISCUSSION THREAD

**Discussion by estherjsong** [ANLAB-KAIST/KENSv3#189] [Project Q&A 2. Transport Layer (TCP)]
![image](https://github.com/ANLAB-KAIST/KENSv3/assets/40757981/9901b35c-d600-4b24-9ba8-992b2aae1c0c)

If the TestAccept_MultipleInterface1 starts, It doesn't get through assertion  `nsIter != appIter->second.fdToDomain.end()'  .
I saw the E_Host.cpp file, but I can't understand why I can't get through that assertion and even what it asserts.

How can i get through it? I'm looking for this 2 days long.

**Comment by Pusnow** [comment]
The error occurs when you have not registered FD.

You should register FDs using `createFileDescriptor`: https://github.com/ANLAB-KAIST/KENSv3/wiki/Tip:-Managing-File-Descriptors
